aws/resource_aws_resource_group_test.go

```diff
@@ -0,0 +1,103 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/resourcegroups"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSResourceGroup_importBasic(t *testing.T) {
```

Comment:
Instead of creating a separate acceptance test that essentially duplicates the regular test, we prefer to add the import `TestStep` to all other acceptance tests. 👍